actor/builtin/storagemarket/storagemarket.go

```diff
@@ -233,12 +233,12 @@ func (sma *Actor) GetTotalStorage(vmctx exec.VMContext) (*big.Int, uint8, error)
 		return nil, errors.CodeError(err), err
 	}
 
-	count, ok := ret.(*big.Int)
+	amt, ok := ret.(*types.BytesAmount)
```

Comment:
I'm rob pike and I approve this go-y name change

Reply:
i literally loled